en/flight_controller/cuav_v5_nano.md

```diff
@@ -92,6 +92,11 @@ make px4_fmu-v5_default
   If you're connecting peripherals that must be powered, you will need to separately power the output using a BEC.
 
 
+## Current Protection
```

Comment:
Yeah fix the header too "over current "

Reply:
Fixed inhttps://github.com/PX4/px4_user_guide/commit/1d8264a4bafe3e8df2e6d9fd5fb099bceeefa827